independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PaySpace",
+    "description": "Our powerful API allows you to access your employee data in order to utilize in your business environment. All supported endpoints in one place and ready to be integrated into your solution.",
+    "contact": {
```

Comment:
@mint-virgil please have your information or your organization information in the contact section. It should include the publisher information.

Reply:
@Amjed-Ayoub I've put in my organization information in here now.

Reply:
the "name" and the "email" Properties are still for "PaySpace". can you please replace with your information or your organization information?

Reply:
Hi there, I've updated these details in the latest commit